File: app/src/main/java/piuk/blockchain/android/data/contacts/ContactsMapStore.java
package piuk.blockchain.android.data.contacts;

import java.util.HashMap;

public class ContactsMapStore {

    private HashMap<String, String> contactsTransactionMap = new HashMap<>();
    private HashMap<String, String> notesTransactionMap = new HashMap<>();

    public HashMap<String, String> getContactsTransactionMap() {
        return contactsTransactionMap;
    }

    public HashMap<String, String> getNotesTransactionMap() {
        return notesTransactionMap;
    }

    public void clearContactsTransactionMap() {
        contactsTransactionMap.clear();
    }

    public void clearNotesTransactionMap() {
        notesTransactionMap.clear();
    }
}

File: app/src/main/java/piuk/blockchain/android/data/contacts/ContactsMapStore.kt
package piuk.blockchain.android.data.contacts

import java.util.*

class ContactsMapStore {

    val contactsTransactionMap = HashMap<String, String>()
    val notesTransactionMap = HashMap<String, String>()

    fun clearContactsTransactionMap() {
        contactsTransactionMap.clear()
    }

    fun clearNotesTransactionMap() {
        notesTransactionMap.clear()
    }
}
